packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=T4Uehk3_wlY}
 ///
+///
+///{@tool sample --template=stateless_widget_scaffold_center.tmpl}
```

Comment:
You don't need the `.tmpl` suffix here. Also, Make sure there's at least one space between the code comment `///` and the comment contents, here and throughout the change.
```suggestion
/// {@tool sample --template=stateless_widget_scaffold_center}
```